packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -517,6 +519,53 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     return response.map(deserializeBlobAndProofs);
   }
 
+  async getBlobsV2(fork: ForkName, versionedHashes: VersionedHashes): Promise<BlobAndProofV2[]> {
+    if (ForkSeq[fork] < ForkSeq.fulu) {
+      throw Error(`engine_getBlobsV2 not available for fork=${fork}`);
+    }
+
+    // retry only after a day may be
+    const GETBLOBS_RETRY_TIMEOUT = 256 * 32 * 12;
```

Comment:
does not look like this is 1 day time which is `60s * 60 * 24`
also if this is in sec, prefer to suffix it with `_SEC`

Reply:
this retry behavior is not something we should require if blobs v2 is mandatory which I think it should be for Fulu